### ISSUE TITLE
论文中simota训练不稳定的现象具体指的是什么

### ISSUE THREAD

**Issue by ianpundar** [meituan/YOLOv6#711]
### Before Asking

- [X] I have read the [README](https://github.com/meituan/YOLOv6/blob/main/README.md) carefully. 我已经仔细阅读了README上的操作指引。

- [X] I want to train my custom dataset, and I have read the [tutorials for training your custom data](https://github.com/meituan/YOLOv6/blob/main/docs/Train_custom_data.md) carefully and organize my dataset correctly; (FYI: We recommand you to apply the config files of xx_finetune.py.) 我想训练自定义数据集，我已经仔细阅读了训练自定义数据的教程，以及按照正确的目录结构存放数据集。（FYI: 我们推荐使用xx_finetune.py等配置文件训练自定义数据集。）

- [x] I have pulled the latest code of main branch to run again and the problem still existed. 我已经拉取了主分支上最新的代码，重新运行之后，问题仍不能解决。

### Search before asking

- [X] I have searched the YOLOv6 [issues](https://github.com/meituan/YOLOv6/issues) and found no similar questions.

### Question

请问论文中说SimOTA导致训练不稳定的现象具体是什么？

**Comment by meituan-gengyifei** [comment]
Hi, thank you for using YOLOv6！The instability mentioned in the paper is mainly manifested in the fact that the model is prone to nan divergence under different batch training parameters.For example, when using the l model to train the coco data set, when the batch is increased to 256, it is easy to have nan in the early stage of training, and often reducing the batch can alleviate this problem. However, this problem cannot be solved by adjusting the batch on different business data sets. In addition to the problem of nan, it is also prone to fluctuations in training indicators, which will affect the final model effect.The reason for this is probably mainly because the logic of SimOTA is to use the training loss to select samples, which is easy to produce a negative cycle when the model accuracy is poor in the early stage of training, resulting in unstable training.

**Comment by ianpundar** [comment]
Thanks!!
